src/2sv/Change.vue

```diff
@@ -27,24 +27,30 @@
 </template>
 
 <script>
-import { find, remove, mfa } from '@/global/mfa';
+import { find, remove, removeWebauthn, mfa, retrieve } from '@/global/mfa'
 
 export default {
   data: () => ({
     requested: {},
   isLastOne: mfa.numVerified === 1
   }),
   computed: {
-    label: vm => vm.requested && vm.requested.label || '2SV'
+    label: vm => vm.requested?.label || '2SV',
   },
   methods: {
     async yes(id) {
-      await remove(id)
+      if (mfa.keys.id === id) {
+        await removeWebauthn(id, mfa.keys.data[0].id)
+      }
+      else {
+        await remove(id)
+      }
 
       this.$router.push('/2sv/removed')
     },
   },
-  created() {
+  async created() {
+    await retrieve()
     this.requested = find(this.$route.params.id)
   },
 }
```

Comment:
I don't think we use this view for webauthn anymore since we removed the "change button" from the card. The only way it would be reached is if someone put the url in manually. Should we remove the code related to webauthn?

Reply:
I think removal of the code can wait for a future release.